public/public.class.php

```diff
@@ -215,15 +211,116 @@ public static function listing(){
    *
    * @since    1.0.0
    */
-  public static function listing_grid(){
+  public static function listing_grid($opts){
+
+    $layout = empty($opts['layout']) ? 1 : $opts['layout'];
+
+    self::$options = get_option( 'purecharity_fundraisers_settings' );
+
+    switch ((int) $layout) {
+      case 1:
+        return self::grid_option_3();
+        break;
```

Comment:
My bad. I was forcing option 3 there so I didn't need to change the shortcode on the admin pane ☕